bin/wp-env-config.sh

```diff
@@ -22,6 +22,7 @@ wp wc customer update 1 --user=1 --billing='{"first_name":"John","last_name":"Do
 
 ## Prepare translation for the test suite
 wp language core install nl_NL
+wp language plugin install woocommerce nl_NL
 wp language plugin install woo-gutenberg-products-block nl_NL
 # We need to run update after installing the langauge to update it to the latest version. Otherwise, new strings won't be available.
```

Comment:
Can you please help me correct the typo for this line, the word 'langauge'?

Reply:
Done